dev/CommonManaged/PlatformConfiguration.cs

```diff
@@ -31,6 +31,38 @@ public enum OSVersion : ushort
     public class PlatformConfiguration
     {
         const OSVersion MaxOSVersion = OSVersion.Redstone2;
+        private const ushort InvalidAPIVersion = 255;
+        private static ushort _currentAPIVersion = InvalidAPIVersion;
+
+        private static bool IsApiContractPresent(ushort version)
+        {
+            return ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", version);
+        }
+
+        public static ushort GetCurrentAPIVersion()
+        {
```

Comment:
Why the complex logic to start at 7 and walk up/down ? Is there a perf concern ? This looks like test code and will run only once, so perhaps a simple walk up would suffice ?